### PR TITLE
[added] Apply custom data- and aria- props to content div

### PR DESCRIPTION
**Pull request by Munter**
Changes proposed:
- Apply `aria-*` and `data-*` props to the modal content div

Acceptance Checklist:
- [x] All commits have been squashed to one.
- [x] The commit message follows the guidelines in `CONTRIBUTING.md`.
- [x] Documentation (README.md) and examples have been updated as needed.
- [x] If this is a code change, a spec testing the functionality has been added.
- [x] If the commit message has [changed] or [removed], there is an upgrade path above.

### CODE REVIEW

**Comment by diasbruno** [comment]
This looks good for aria attributes.

File: lib/components/ModalPortal.js [hunk 1]
```diff
@@ -18,6 +19,18 @@ var CLASS_NAMES = {
   }
 };
 
+function getCustomProps(props) {
```
Comment (diasbruno) [inline lib/components/ModalPortal.js]: Maybe move this function to a helper?

**Comment by afercia** [comment]
It would be great to have the ability to add ARIA attributes on the modal, this would be necessary for example when the modal content is mainly text:

screen shot 2017-05-11 at 16 58 53

The text inside the modal won't be announced because screen readers enter in focus mode when `role=dialog` is set., Using `aria-describedby` to target the text and optionally `aria-labelledby` to target the visible heading would solve the accessibility issue.

**Comment by diasbruno** [comment]
Hi @Munter, we have `v2.0.0` available now. We could make this PR to allow the user to pass `data-*` attributes and later we can think about adding `aria-*`. Any thoughts?

**Comment by Munter** [comment]
@diasbruno Thanks for the heads up. I switched to react-a11y-modal because this was a blocker to me. I don't have the bandwidth to port this to v2

**Comment by diasbruno** [comment]
@Munter no problem. Thank you.

**Comment by diasbruno** [comment]
@Munter Do you mind if I slice your PR and add to `react-modal`?

**Comment by Munter** [comment]
Take and modify whatever you need from this. It would be a fine addition to the library to get those features in

**Comment by WickyNilliams** [comment]
Is this no longer supported?

**Comment by diasbruno** [comment]
@WickyNilliams `aria-` attributes is available. See [https://github.com/reactjs/react-modal#additional-aria-attributes](https://github.com/reactjs/react-modal#additional-aria-attributes).

**Comment by WickyNilliams** [comment]
ah... i specifically wanted data-* attributes, as hooks for targeting elements in tests

**Comment by diasbruno** [comment]
One option is place those attributes in a wrapper on the content `<div />` or something else.

```jsx
<Modal ...>
    <div data-x=... >...</div>
</Modal>
```

**Comment by diasbruno** [comment]
@WickyNilliams Can you open a new issue to make it easy to track?

**Comment by WickyNilliams** [comment]
Yeah that's what I did eventually, but not ideal.

Sure thing! I'll make one now

**Pull request closed without merging**